deployment/ansible/group_vars/all.example

```diff
@@ -90,3 +90,9 @@ monit_allow_hosts: []
 monit_allow_user: 'test'
 monit_allow_password: 'test'
 monit_enable_web_access: yes
+
+## IP addresses used to configure the firewall
+ip_addresses_app:
```

Comment:
Turns out to not be so easy. I tried several options, none of which worked in this context. I'm going to pass on it for now and merge this to test in staging.